webapp/src/components/ResourceList.tsx

```diff
@@ -216,7 +218,9 @@ export default class ResourceList extends React.Component<IResourceListProps, IS
           <p>A resource is a collection of files on HydroShare, a place for sharing code and water data. These files can be code (e.g. Python or R), data (e.g. .csv, .xlsx, .geojson), or any other type of file.</p>
           <p>The list below shows the resources that exist in HydroShare and in JupyterHub. Resources only in HydroShare can be synced to JupyterHub, and then you can run code and edit data. All changes should be made
              in JupyterHub and then synced to HydroShare. Think of JupyterHub as your workspace and HydroShare are your sharing or archival space. </p>
-          <p>To begin, click the New Resource button to create a new resource or click on an existing resource in the list to view files in that resource.</p> 
+          <p>To begin, click the <b>New Resource</b> button to create a new resource or click on an existing resource in the list to view files in that resource.</p> 
+          <p><b>Delete: </b> This will delete a resource from your workspace and from HydroShare. Please save any files you want to your desktop before deleting as all of your work will be lost.</p>
+          <p><b>Archive: </b> This will delete a resource from your workspace but save it in HydroShare. Please manually transfer from your workspace any remaining files you'd like to save to HydroShare before archiving as all of your files in your workspace will be lost.</p>
```

Comment:
Wait this archive description is a little confusing.. It won't actually save it in HydroShare, right? The user still has to do that? If that is the case then I would say you should say something along the lines of "This will delete a resource from your workspace but not from HydoShare" for that first sentence instead of implying that the archive action actually does some saving.

Reply:
Maybe if we just call it "Remove from Workspace", then we don't even need descriptions here. We should have the explanations in the delete confirmation modals though.

Reply:
sure, "remove from workspace" will work